pex/bin/pex.py

```diff
@@ -457,9 +457,10 @@ def interpreter_from_options(options):
     return interpreter
 
 
-def build_pex(args, options, resolver_option_builder):
-  with TRACER.timed('Resolving interpreter', V=2):
-    interpreter = interpreter_from_options(options)
+def build_pex(args, options, resolver_option_builder, interpreter=None):
```

Comment:
I'm a little worried about the implicit lack of interpreter setup with this proposed change - seems to me you'd either have to do that manually or call `interpreter_from_options` anyway.

so.. why not just pass `options.python` to `build_pex(options, ...)` with a fully qualified path to the preferred interpreter to achieve the same behavior?

this would provide interpreter setup for ~free and works in today's API.